tests/cache_sorted_set.rs

```diff
@@ -73,23 +73,23 @@ fn compare_by_first_entry(a: &(String, f64), b: &(String, f64)) -> std::cmp::Ord
 async fn sorted_set_put_elements_happy_path() {
     async fn test_put_elements_happy_path(
         client: Arc<CacheClient>,
-        cache_name: String,
+        cache_name: &String,
         to_put: impl IntoSortedSetElements<String> + Clone,
     ) {
-        let sorted_set_name = "sorted-set-".to_string() + &Uuid::new_v4().to_string();
+        let sorted_set_name = format!("sorted-set-{}", Uuid::new_v4());
         let result = client
-            .sorted_set_fetch_by_score(cache_name.clone(), sorted_set_name.clone(), Ascending)
+            .sorted_set_fetch_by_score(cache_name, sorted_set_name.as_str(), Ascending)
```

Comment:
this is a supernit but I would recommend `&sorted_set_name` here if it works. and if it doesn't work then I'd doublecheck our API sig and see if we think it *should* work :)

Reply:
Sorry, I missed that reply. I don't quite understand though, what error do you get if you just do `&sorted_set_name` on the function arg?

Don't block on this though. merge if you're confident.